src/main/java/de/retest/recheck/ui/diff/ElementDifference.java

```diff
@@ -46,6 +46,7 @@ public ElementDifference( final Element element, final AttributesDifference attr
 		this.expectedScreenshot = expectedScreenshot;
 		this.actualScreenshot = actualScreenshot;
 		this.childDifferences.addAll( childDifferences );
+		Collections.sort( this.childDifferences );
```

Comment:
What's the benefit?

Reply:
It does not sort in place but creates a new list (which you essentially do here) and allow for immutability

Reply:
This is one of the short-comings of Java: This list is immutable (i.e. final) already. Because of `beforeMarshal`, what you want cannot be enforced, so I don't see a benefit.